x-team/xp-ui
# Create JobsPageLayout component

**Release Type:** *Non-breaking change*

Fixes https://x-team-internal.atlassian.net/browse/XP-3364

## Description

Create `JobsPageLayout` component

## Checklist


## Steps to Test or Reproduce

1. Run `npm install`
1. Run `npm start`
1. Go to http://localhost:9001/?path=/story/ui-components-xp-registration-jobspagelayout--default-usage-full-composition (notice that there's some temporary placeholder for components that doesn't exist yet, also the `JobsGrid` will look better in the deliverable of https://x-team-internal.atlassian.net/browse/XP-3365)
1. Observe responsiveness options (test it in smaller viewports)
1. Observe variation with Loading under Debug navigation group
1. Observe variations with debug components (under Debug navigation group)

## Impacted Areas in Application

- `JobsPageLayout` was created
- `ApplicantScreen` got small updates

## Screenshots

<img width="1678" alt="Screen Shot 2019-10-08 at 12 58 33" src="https://user-images.githubusercontent.com/131859/66415643-7103f700-e9d2-11e9-84c0-410337fa3596.png">

![2019-10-08 13 02 37](https://user-images.githubusercontent.com/131859/66415641-7103f700-e9d2-11e9-8aba-41fbac7f4adc.gif)

<img width="1676" alt="Screen Shot 2019-10-08 at 14 10 37" src="https://user-images.githubusercontent.com/131859/66417041-8a5a7280-e9d5-11e9-943c-ce29bf5aa06a.png">
